### PR TITLE
Include CRD Group in Filename

### PR DESCRIPTION
**Pull request by jacksgt**
This avoids conflicts between CRDs which have the same short name but are coming from a different group (e.g. `Project/config.openshift.io/v1` vs. `Project/project.openshift.io/v1`)

### CODE REVIEW

**Comment by jacksgt** [comment]
CI should be fixed now.

File: src/crd2jsonschema.sh [hunk 1]
```diff
@@ -65,7 +73,9 @@ function get_jsonschema_file_name()
     crd_kind="$(get_crd_kind "$crd")" || exit 1
     local crd_version
     crd_version="$(get_crd_version "$crd")" || exit 1
-    echo "${crd_kind}_${crd_version}.json"
+    local crd_group
+    crd_group="${get_crd_group "$crd")" || exit 1
```
Comment (tricktron) [inline src/crd2jsonschema.sh]:
```suggestion
    crd_group="$(get_crd_group "$crd")" || exit 1
```
Hello @jacksgt. Thank you for the contribution. This typo should make it work.

**Review by tricktron — CHANGES_REQUESTED** [review]

File: src/crd2jsonschema.sh (hunk 1 above)
Comment (jacksgt) [reply src/crd2jsonschema.sh]: Oh indeed, well spotted! Fixed.

**Comment by jacksgt** [comment]
Man you're tests are quite comprehensive :sweat_smile: 
I fixed the integration test as well.

**Comment by tricktron** [comment]
> Man you're tests are quite comprehensive 😅 I fixed the integration test as well.

Yeah, I probably test here a bit too implementation specific. Thank you for the fixes 👍. I'll merge so that a new docker image should be built and will create a release.

**Review by tricktron — APPROVED** [review]

**Comment by tricktron** [comment]
@jacksgt Release done and new docker images are pushed. Cheers.

**Comment by jacksgt** [comment]
@tricktron Awesome thanks for the quick action - it's very much appreciated!

**Pull request merged**